src/cpu.rs

```diff
@@ -72,7 +72,7 @@ impl Cpu {
             };
         }
 
-        macro_rules! logical_non_immediate {
+        macro_rules! alu_non_immediate {
```

Comment:
I'm assuming this stands for arithmetic logic unit? Good naming :)

Reply:
yup. Thanks.